modules/imap/handler_modules.php

```diff
@@ -1933,6 +1933,12 @@ public function process() {
             $this->out('imap_accounts', $this->user_config->get('imap_servers'), array());
             $this->out('show_pagination_links', $this->user_config->get('pagination_links_setting', true));
             $this->out('auto_advance_email_enabled', $this->user_config->get('auto_advance_email_setting', true));
+
+            $parts = explode('_', 'imap_' . $form['imap_server_id'] . '_' . $form['folder'], 3);
+            $details = Hm_IMAP_List::dump($parts[1]);
```

Comment:
Why don't you simply use $form['imap_server_id'] here? I don't see a reason for the explode...